downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,237 @@
+
+## Update Ballerina
+
+Update your current Ballerina installation directly to 2201.7.0 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/) as follows.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+2. Run `bal dist update` to update to this latest distribution.
+
+## Install Ballerina
+
+If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
+
+## Backward-incompatible changes
+
+- A bug that allowed using a function reference of a `non-isolated` function type in a function or method call expression within an `isolated` function or method has been fixed.
+
+    ```ballerina
+    type Employee record {|
+        string name;
+        int id;
+        string department;
+    |};
+
+    isolated function createEmployee(string[] name,
+            function () returns int idFunction,
+            string department)
+            returns Employee => {
+        name: string:'join(" ", ...name),
+        // Compilation error now, since `idFunction` is not `isolated`.
+        id: idFunction(),
+        department
+    };  
+    ```
+  
+- A bug that caused the broader type to be used instead of the singleton type when a constant is used as a type descriptor has been fixed.
+
+  For example, the code below may result in a compilation error now since `num`, which is of type `CONST` can only be integer `8`.
+
+    ```ballerina
+    const int CONST = 1 + 7;
+    CONST num = 2; // Compilation error now.
+    ```
+
+- A bug that resulted in compilation errors not being logged for duplicate keys via computed name fields of a mapping constructor in a mapping constant declaration has been fixed. This previously resulted in a panic at runtime instead.
+
+    ```ballerina
+    const MESSAGE = "message";
+    const map<string> HTTP_OK = {httpCode: "200", message: "OK", [MESSAGE] : "BAD REQUEST"}; // Compilation error now.
+    ```
+
+- Fixed a bug in the configurable TOML syntax validation for the module structure. The error message thrown for an invalid TOML module structure is now improved to provide the variable name.
+
+    For example, consider a non-default module named `foo.bar`, which contains the following configurable variables.
+
+    ```ballerina
+    configurable int intVar = ?;
+    configurable string stringVar = ?;
+    ```
+
+    If an invalid TOML structure is found for the `foo.bar` module in the `Config.toml` file, it throws an error with the following message.
+
+    ```
+    [Config.toml:(1:1,2:23)] invalid TOML structure found for module ’foo.bar’. with variable 'intVar'. Please provide the module name as '[foo.bar]'
+    [Config.toml:(1:1,2:23)] invalid TOML structure found for module ‘foo’.bar. with variable 'stringVar'. Please provide the module name as ‘[foo.bar]’
+    ```
+
+## Platform updates
```

Comment:
Is this the first section intentionally?

Reply:
Yes. Is there a reason to not make it the first section? @MaryamZi

Reply:
Because we usually have lang updates first, then stdlib, etc.

Reply:
Added the topic at the top because to highlight it as it is one of the main features in this release and also because it affects the entire platform in general.